src/components/register/register.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { instanceOf } from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import {
+  FormControl,
+  Image,
+  FormLabel,
+  InputGroup,
+  Input,
+  InputRightElement,
+  Button,
+  Flex,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Flex' is defined but never used.